### ISSUE TITLE
show "og:image:alt" on preview card

### ISSUE THREAD

**Issue by tribela** [mastodon/mastodon#18555]
### Pitch

Opengraph can have `og:image:alt` attribute for image alternate text.
It could be replace current `alt=""` on mastodon's preview card

### Motivation

To help low vision people, Also helpful for who using translator.
If there are already given text for `alt` attr, why not to using it

**Comment by lupinia** [comment]
Agreed 100%; I recently implemented OpenGraph tags on my own website, specifically making a point to include `og:image:alt` for posting links to my photography on Mastodon, so the fact that this isn't implemented on the Mastodon side is really disappointing.

**Comment by tcitworld** [comment]
Closed by https://github.com/mastodon/mastodon/pull/26184 ?
